scapy/arch/windows/__init__.py

```diff
@@ -323,7 +327,7 @@ def get_windows_if_list():
                             'GUID', 'MacAddress'])
     return [
         iface for iface in
-        (dict(zip(['name', 'win_index', 'description', 'guid', 'mac'], line))
+        (dict(list(zip(['name', 'win_index', 'description', 'guid', 'mac'], line)))
```

Comment:
This change is not useful as `dict(zip( .. ))` works fine on both Python2 and Python3